### PR TITLE
improve naming of `access` values when defining `-jpms-module-info`

### LINKED ISSUE #4164: improve naming of `access` values when defining `-jpms-module-info`
Currently it only works if you use the based 10 integer value (32, 4096, 32768) instead of either a literal name of the hex values defined in the spec (0x0020, 0x1000, 0x8000).

### PR DESCRIPTION
**Pull request by rotty3000**
Fixes #4164

Signed-off-by: Raymond Augé <raymond.auge@liferay.com>

### CODE REVIEW

File: biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java [hunk 1]
```diff
@@ -76,6 +78,46 @@
  */
 public class JPMSModuleInfoPlugin implements VerifierPlugin {
 
+	enum Access {
+		CLOSED(
```
Comment (bjhargrave) [inline biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java:82]: I hope to god the next Eclipse version fixes this idiotic code formatting bug...

File: biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java [hunk 2]
```diff
@@ -76,6 +78,46 @@
...
+			0),
+		OPEN(ACC_OPEN),
+		SYNTHETIC(ACC_SYNTHETIC),
+		MANDATED(ACC_MANDATED);
+
+		public static Access parse(String input) {
+			switch (input) {
+				case "OPEN" :
+				case "open" :
+				case "0x0020" :
```
Comment (bjhargrave) [inline biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java:92]: Isn't this a bit restrictive? `0x20` wont work. You probably need two paths: string and numbers.

Comment (rotty3000) [reply biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java:92]: k

Comment (rotty3000) [reply biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java:92]:
```
				default :
					int parsedValue = Integer.parseInt(input);
					return Arrays.stream(values())
						.filter(a -> a.getValue() == parsedValue)
						.findFirst()
						.orElse(CLOSED);

```
?

Comment (bjhargrave) [reply biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java:92]: I think you want `Integer.decode` which can handle hex strings and octal. But you should catch any exception from decode and map to an error.

Comment (bjhargrave) [reply biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java:92]: Also the orElse case should also be an error, no?

File: biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java (hunk 1 above)
Comment (rotty3000) [reply biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java:83]: :)

**Pull request merged**